server/controllers/articleController.js

```diff
@@ -17,9 +18,21 @@ export default class ArticleController {
   static createArticle(req, res) {
     const { title, body, description } = req.body;
     const imgUrl = (req.file ? req.file.secure_url : '');
-    Article.create({
-      title, body, userId: req.user.id, description, slug: `${slug(title)}-${uuid()}`, imgUrl
-    }).then(article => res.status(201).json({ message: 'article created successfully', success: 'true', article }))
+    getTagIds(req, res).then((tagIds) => {
+      Article.create({
```

Comment:
This change here gets the ids of the tags sent along with the request. Then the article is created and the tagged. Then, the article is returned with the tags.